### PR TITLE
Prepare for initial gem release

### PR DESCRIPTION
**Pull request by ms-ati**
Add Rakefile with "bundler/gem_tasks"
Add instructions for releasing to README.md
Add authors and emails to gemspec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,29 @@ https://github.com/panorama-ed/memo_wise. This project is intended to be a safe,
 welcoming space for collaboration, and contributors are expected to adhere to
 the [code of conduct](https://github.com/panorama-ed/memo_wise/blob/main/CODE_OF_CONDUCT.md).
 
+## Releasing
+
+To make a new release of `MemoWise` to Rubygems, first install the release
```
Comment (jemmaissroff) [inline README.md:160]: Would it make sense to link RubyGems here? Or maybe the memo_wise RubyGems page once it's up?

**Review by jemmaissroff — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JacobEvelyn) [reply README.md:160]: While we're talking about this line, tiny nitpick: `Rubygems` -> `RubyGems`

File: README.md [hunk 2]
```diff
@@ -155,6 +155,29 @@ https://github.com/panorama-ed/memo_wise. This project is intended to be a safe,
...
+dependencies:
+
+```shell
+bundle config set --local with 'release'
```
Comment (JacobEvelyn) [inline README.md:164]: I'm not sure I understand—why is this necessary?

Comment (JacobEvelyn) [reply README.md:164]: Ohh, this is so that `rake release` works?

File: README.md [hunk 3]
```diff
@@ -155,6 +155,29 @@ https://github.com/panorama-ed/memo_wise. This project is intended to be a safe,
...
+1. Update `CHANGELOG.md`:
+   - Add an entry for the upcoming version _x.y.z_
+   - Move content from _Unreleased_ to the upcoming version _x.y.z_
+   - Commit with title `Update CHANGELOG.md for x.y.z`
+
+2. Update `lib/memo_wise/version.rb`
+   - Replace with upcoming version _x.y.z_
+   - Commit with title `Bump version to x.y.z`
+
+3. `rake release`
```
Comment (JacobEvelyn) [inline README.md:179]: Does this need to be `bundle exec rake release`?

File: README.md (hunk 2 above)
Comment (JacobEvelyn) [inline README.md:164]: Do we need to undo this `bundle config set` afterwards?

File: README.md (hunk 1 above)
Comment (ms-ati) [reply README.md]: Updated to fix capitalization and add link!

File: README.md (hunk 2 above)
Comment (ms-ati) [reply README.md:165]: Yes -- I've added some clarification above, is this good now?

File: README.md (hunk 3 above)
Comment (ms-ati) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (ms-ati) [reply README.md:165]: I would not think so? This just means that your local `bundle config` now installs the group `release` for this `Gemfile` -- what would be the benefit of undoing that?

**Review by jemmaissroff — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (JacobEvelyn) [reply README.md:165]: nvm, you're right.

**Review by JacobEvelyn — APPROVED** [review]
